### PR TITLE
DHFPROD-5984: Filter on sourceName and sourceType as hub facets.

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/config/hub-properties.config.js [hunk 1]
```diff
@@ -30,6 +30,22 @@ const hubPropertiesConfig = [
     propertyPath: 'createdByStep',
     entityTypeId: ' '
   },
+  {
+    facetName: "sourceName",
+    displayName: "SourceName",
```
Comment (rjrudin) [inline marklogic-data-hub-central/ui/src/config/hub-properties.config.js:35]: Can there be spaces in a "displayName"? We'd want this to be as user-friendly as possible, so "Source Name" would be better than "SourceName".

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/config/hub-properties.config.js]: Yes there can be spaces, thanks for noticing that.

**Review by Sanjeevani19 — CHANGES_REQUESTED** [review]
@timur-isangulov The UI breaks when shifting between entities, please look into it
https://drive.google.com/file/d/1DNsnYTwJEmsmy-hO-JsalxMwY8NrhZx7/view

**Comment by timur-isangulov** [comment]
> @timur-isangulov The UI breaks when shifting between entities, please look into it
> https://drive.google.com/file/d/1DNsnYTwJEmsmy-hO-JsalxMwY8NrhZx7/view
@Sanjeevani19 most likely backend changes are missing, can you please redeploy project using latest backend.

**Review by Sanjeevani19 — APPROVED** [review]

**Review by rahulvudutala — APPROVED** [review]

**Review by xnikhil08 — APPROVED** [review]

**Pull request merged**